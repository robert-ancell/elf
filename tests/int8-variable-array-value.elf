int8 v = [1, 2, 3]
