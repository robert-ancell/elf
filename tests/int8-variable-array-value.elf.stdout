Line 1:
int8 v = [1, 2, 3]
     ^
Variable is of type int8, but value is of type uint8[]
